packages/flame/lib/src/components/parallax_component.dart

```diff
@@ -73,14 +78,15 @@ class ParallaxComponent extends PositionComponent {
     )..parallax = parallax;
   }
 
-  @mustCallSuper
```

Comment:
Why remove this line?

Reply:
Because when I used this on my own `ParallaxComponent` when I found this bug I realized one might want to do their own logic here and then who are we to stop them? :man_shrugging:

Reply:
Yeah, but isn't the super logic important on this case? Or is this so simple that would make sense for the user to skip super?

Reply:
It is important if you have children, otherwise it will just get in your way if you want to handle this yourself.
But I'm up for any, there are cons and pros with having and not having this line.
Maybe it is better to keep it and let the advanced users just ignore the warning?

Reply:
If there are use cases for ignoring it, I am ok on removing the mustCallSuper, and just add a note on the docs for when you have children, you may face issues if super isn't called? Wdyt?

Reply:
I re-added it already :sweat_smile: 
I think the users that want to implement their own logic there will be advanced enough to know what they are doing, there isn't as much of a use-case anymore now when I changed the logic of that method in this PR.

Reply:
Hahhahah, alright, up to you!